selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/plainjava/RunPlainJavaProjectTest.java

```diff
@@ -53,7 +52,7 @@
 
 /** @author Aleksandr Shmaraev */
 public class RunPlainJavaProjectTest {
-  private static final String PROJECT_NAME = NameGenerator.generate("RunningPlainJavaProject", 4);
+  private static final String PROJECT_NAME = "run-plain-java-project";
```

Comment:
it resolves the problem about importing Simple project in jdt.ls workspace. Because `.project` file contains name of the project and here, when the name is generated, it isn't equal.

Reply:
Actually, the eclipse workspace project name should not make a difference, (i.e. we should not depend on it), but I guess since we're checking for log messages :-(